OracleGoldenGate/Dockerfile

```diff
@@ -0,0 +1,82 @@
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# Copyright (c) 2014-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE GOLDENGATE DOCKER PROJECT
+# --------------------------------
+#
+# Dockerfile for Oracle GoldenGate
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+#
+# From http://www.oracle.com/technetwork/middleware/goldengate/downloads/index.html
+#  Download the file for "Oracle GoldenGate 12.x.x.x.x for Oracle on Linux x86-64"
+#    - fbo_ggs_Linux_x64_shiphome.zip
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+#
+# Execute the "buildDockerImage.sh" shell script, with the name of the downloaded OGG filename.
+# Example:
+#      $ ./buildDockerImage.sh ~/Downloads/fbo_ggs_Linux_x64_shiphome.zip
+#
+
+ARG         BASE_IMAGE=container-registry.oracle.com/database/instantclient:12.2.0.1
```

Comment:
FROM should be the first line in the Dockerfile. Just a matter of standards and code format we are following